protocols/data_processor.py

```
@@ -4,101 +4,106 @@
                 try:
-                    with open(file_path, 'r') as f:
+                    with open(file_path, "r") as f:
                         reader = csv.reader(f)
                         row_count = sum(1 for row in reader)
                         total_records += row_count
                         processed_count += 1
-                        insights.append(f"{filename}: CSV with {row_count} rows")
+                        insights.append(
+                            f"{filename}: CSV with {row_count} rows"
+                        )
                 except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)